### ISSUE TITLE
Page Builder - enable the editor on New pages

### ISSUE THREAD

**Issue by leandrocp** [BeaconCMS/beacon_live_admin#131]
Users have the option to edit a page template either using the Code Editor (default view) or the Visual Editor:

<img width="1360" alt="Screenshot 2024-06-12 at 4 21 00 PM" src="https://github.com/BeaconCMS/beacon_live_admin/assets/36407/6515a5c4-dbd3-4801-89de-482c49bff2ed">

But they don't have the option to use the Visual Editor when creating a new page:

<img width="1376" alt="Screenshot 2024-06-12 at 4 21 23 PM" src="https://github.com/BeaconCMS/beacon_live_admin/assets/36407/54976137-955a-4aa3-a7a7-cea9c3517ff7">

We want to add that "Visual Editor" button on the "Create Page" to allow users creating new pages from scratch using the visual editor but we need to double check if the visual editor works with a blank page, ie: if users can drag and drop components into the page, edit attributes, and save the changes.

In other words, users should be able to create and publish pages only using the visual editor.

**Comment by leandrocp** [comment]
Closed by https://github.com/BeaconCMS/beacon_live_admin/pull/172
